apertium-uzb.uzb.lexc

```diff
@@ -40745,3 +40800,5557 @@ Amerika% Qoʻshma% Shtatlari:Amerika% Qoʻshma% Shtatlari  NP-TOP ; ! "" ! El++
+Valentina:Valentina  NP-ANT-F ; ! "" ! El++
+Valeriya:Valeriya  NP-ANT-F ; ! "" ! El++
+Viktoriya:Viktoriya  NP-ANT-F ; ! "" ! El++
+Xirodiya:Xirodiya  NP-ANT-F ; ! "" ! El++
+Xojar:Xojar  NP-ANT-F ; ! "" ! El++
+Xotin:Xotin  NP-ANT-F ; ! "" ! El++
+Xurriyat:Xurriyat  NP-ANT-F ; ! "" ! El++
+Yekaterina:Yekaterina  NP-ANT-F ; ! "" ! El++
+Yelena:Yelena  NP-ANT-F ; ! "" ! El++
+Yelizaveta:Yelizaveta  NP-ANT-F ; ! "" ! El++
+Yevgeniya:Yevgeniya  NP-ANT-F ; ! "" ! El++
+Yuliya:Yuliya  NP-ANT-F ; ! "" ! El++
+Zavqiya:Zavqiya  NP-ANT-F ; ! "" ! El++
+Zubayda:Zubayda  NP-ANT-F ; ! "" ! El++
+Zuxra:Zuxra  NP-ANT-F ; ! "" ! El++
+
+Aartsen:Aartsen  NP-COG-MF ; ! "" ! El++
+Abazovich:Abazovich  NP-COG-MF ; ! "" ! El++
+Abbos:Abbos  NP-COG-MF ; ! "" ! El++
+Abdulayev:Abdulayev  NP-COG-MF ; ! "" ! El++
+Abdulayeva:Abdulayeva  NP-COG-MF ; ! "" ! El++
+Abdulganiyev:Abdulganiyev  NP-COG-MF ; ! "" ! El++
+Abdulgʻaniyev:Abdulgʻaniyev  NP-COG-MF ; ! "" ! El++
+Abdulganiyeva:Abdulganiyeva  NP-COG-MF ; ! "" ! El++
+Abdulgʻaniyeva:Abdulgʻaniyeva  NP-COG-MF ; ! "" ! El++
+Abdurahmanov:Abdurahmanov  NP-COG-MF ; ! "" ! El++
+Abdurahmanova:Abdurahmanova  NP-COG-MF ; ! "" ! El++
+Abduraimov:Abduraimov  NP-COG-MF ; ! "" ! El++
+Abduraimova:Abduraimova  NP-COG-MF ; ! "" ! El++
+Abe:Abe  NP-COG-MF ; ! "" ! El++
+Abibullayev:Abibullayev  NP-COG-MF ; ! "" ! El++
+Abibullayeva:Abibullayeva  NP-COG-MF ; ! "" ! El++
+Abilov:Abilov  NP-COG-MF ; ! "" ! El++
+Abilova:Abilova  NP-COG-MF ; ! "" ! El++
+Abisov:Abisov  NP-COG-MF ; ! "" ! El++
+Abisova:Abisova  NP-COG-MF ; ! "" ! El++
+Abisovich:Abisovich  NP-COG-MF ; ! "" ! El++
+Abiyev:Abiyev  NP-COG-MF ; ! "" ! El++
+Abiyeva:Abiyeva  NP-COG-MF ; ! "" ! El++
+Abizov:Abizov  NP-COG-MF ; ! "" ! El++
+Abizova:Abizova  NP-COG-MF ; ! "" ! El++
+Abkadirov:Abkadirov  NP-COG-MF ; ! "" ! El++
+Abkadirova:Abkadirova  NP-COG-MF ; ! "" ! El++
+Ablalimov:Ablalimov  NP-COG-MF ; ! "" ! El++
+Ablalimova:Ablalimova  NP-COG-MF ; ! "" ! El++
+Ablamitov:Ablamitov  NP-COG-MF ; ! "" ! El++
+Ablamitova:Ablamitova  NP-COG-MF ; ! "" ! El++
+Ablatipov:Ablatipov  NP-COG-MF ; ! "" ! El++
+Ablatipova:Ablatipova  NP-COG-MF ; ! "" ! El++
+Ablayev:Ablayev  NP-COG-MF ; ! "" ! El++
+Ablazov:Ablazov  NP-COG-MF ; ! "" ! El++
+Ablazova:Ablazova  NP-COG-MF ; ! "" ! El++
+Ablet:Ablet  NP-COG-MF ; ! "" ! El++
+Ablezgov:Ablezgov  NP-COG-MF ; ! "" ! El++
+Ablezgova:Ablezgova  NP-COG-MF ; ! "" ! El++
+Ablimit:Ablimit  NP-COG-MF ; ! "" ! El++
+Abliz:Abliz  NP-COG-MF ; ! "" ! El++
+Abramovich:Abramovich  NP-COG-MF ; ! "" ! El++
+Abrasevich:Abrasevich  NP-COG-MF ; ! "" ! El++
+Absalamov:Absalamov  NP-COG-MF ; ! "" ! El++
+Absalamova:Absalamova  NP-COG-MF ; ! "" ! El++
+Abselamov:Abselamov  NP-COG-MF ; ! "" ! El++
+Abselamova:Abselamova  NP-COG-MF ; ! "" ! El++
+Acimambetov:Acimambetov  NP-COG-MF ; ! "" ! El++
+Acimambetova:Acimambetova  NP-COG-MF ; ! "" ! El++
+Acimovich:Acimovich  NP-COG-MF ; ! "" ! El++
+Adagamov:Adagamov  NP-COG-MF ; ! "" ! El++
```

Comment:
There's a special lexicon for names ending in -ov and -ova in other Turkic transducers (check e.g. kaz and tat).  Could you copy that way of doing this?  Same for -vich, -ovna, etc.

Reply:
Good, I was thinking of having a separate lexicons for that, gonna copy from kaz(or tat) then. For now, I just wanted to have the same tag as in tur.lexc, later regroup again.  I also am planning to import ANT-M and ANT-Fs from Uzbek names book.